src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 96px;
+  height: 116px;
+  margin: 22px;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Currently, the `.stars` class has a margin defined that adds space on all sides. Consider only adding margin to the top or bottom to avoid potential margin collapse issues.